src/ray/gcs/gcs_server/gcs_actor_scheduler.cc

```diff
@@ -440,6 +440,18 @@ void RayletBasedActorScheduler::HandleWorkerLeaseReply(
   // gcs_actor_manager will reconstruct it again.
   auto node_id = NodeID::FromBinary(node->node_id());
   auto iter = node_to_actors_when_leasing_.find(node_id);
+  auto kill_worker = [&reply, this]() {
+    auto &worker_address = reply.worker_address();
+    if (!worker_address.raylet_id().empty()) {
+      auto cli = core_worker_clients_.GetOrConnect(worker_address);
+      rpc::KillActorRequest request;
+      // Set it to be Nil() since it hasn't been setup yet
+      request.set_intended_actor_id(ActorID::Nil().Binary());
+      request.set_force_kill(true);
+      request.set_no_restart(true);
+      RAY_UNUSED(cli->KillActor(request, nullptr));
+    }
+  };
```

Comment:
Add a blank line to make the code look cleaner?

Reply:
Sorry I missed this comment. :(